### PR TITLE
fix: Lint everything

### PR DESCRIPTION
**Pull request by smartcontracts**
## Description
We didn't run `yarn lint:fix`

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

File: packages/x-domain/prettier-config.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (tynes) [inline packages/x-domain/prettier-config.json]: Is there any way that this could be part of `@eth-optimism/dev`?

Comment (smartcontracts) [reply packages/x-domain/prettier-config.json]: Possibly yes, but I haven't looked into the implications of doing it this way.

Comment (smartcontracts) [reply packages/x-domain/prettier-config.json]: I'll add a ticket for it on `@eth-optimism/dev` and handle this in another PR.

Comment (smartcontracts) [reply packages/x-domain/prettier-config.json]: NVM -- just fixed this here instead :-)

**Pull request merged**